src/task/ReceiveVideoStreamIndexTask.ts

```diff
@@ -164,7 +164,15 @@ export default class ReceiveVideoStreamIndexTask
       return;
     }
 
-    this.context.videosToReceive = videoDownlinkBandwidthPolicy.chooseSubscriptions();
+    const videoSubscriptionIdSet = videoDownlinkBandwidthPolicy.chooseSubscriptions();
+    this.context.videosToReceive = videoSubscriptionIdSet.truncate(
+      this.context.videoSubscriptionLimit
+    );
+
+    if (videoSubscriptionIdSet.size() > this.context.videosToReceive.size()) {
+      this.logger.warn(`video receive limit exceeded. Limiting the videos to ${this.context.videosToReceive.size()}`);
```

Comment:
Can we make this log a bit more verbose/helpful and make it clear that builders should change their application code. Anecdotally, some times builders see a log like this and think its a bug on our end.

Reply:
Seems like we should recommend against using AdaptiveProbe? since that will run into this log quite often?

Reply:
imo we should just support in priority-based downlink policy but i guess we can't as priority-based does not work in FF :(

Reply:
Yeah, I can have it set to default as 25 in the protobuf and the AudioVideoControllerState

Reply:
Updated the comment with additional message.

```
video receive limit exceeded. Limiting the videos to 25.
Please consider using AllHighestVideoBandwidthPolicy or VideoPriorityBasedPolicy along 
with choosRemoteVideoSources api to select the video sources to be displayed.
```